packages/codemod/types/paths.types.ts

```diff
@@ -1,3 +1,3 @@
-import { CORE_IMPORT_PATH, CORE_NEXT_IMPORT_PATH } from "../src/consts";
+import { CORE_IMPORT_PATH, CORE_NEXT_IMPORT_PATH, CORE_TYPES_IMPORT_PATH } from "../src/consts";
 
-export type VibeCorePath = typeof CORE_IMPORT_PATH | typeof CORE_NEXT_IMPORT_PATH;
+export type VibeCorePath = typeof CORE_IMPORT_PATH | typeof CORE_NEXT_IMPORT_PATH | typeof CORE_TYPES_IMPORT_PATH;
```

Comment:
I think I made a mistake here
Should it be this instead?:
```suggestion
export type VibeCorePath = CORE_IMPORT_PATH | CORE_NEXT_IMPORT_PATH | CORE_TYPES_IMPORT_PATH;
```